### DISCUSSION TITLE
Ability to add new filetype to Datahub File export plugin

### DISCUSSION THREAD

**Discussion by amarjeet6694** [pimcore/pimcore#9232] [Q&A / Support]
I am doing some R&D on **Datahub File Export** plugin and is trying to add a new _filetype_ 'XLSX' (along-with already added CSV, JSON and XML).
I have followed the steps as given in the [documentation](https://pimcore.com/docs/data-hub-file-export/current/Customize_and_Extend.html#page_Additional-Exporter-Types). But was unable to see new _filetype_ being added in the dropdown

When I checked the code(at _vendor/pimcore/data-hub-file-export/src/Resources/public/js/configuration/fileExport/configItem.js_) I found that we have hard-coded the store values for _filetype_. 
```
{
	fieldLabel: t("plugin_pimcore_datahub_file_export_delivery_destination_filetype"),
	xtype: "combo",
	name: "filetype",
	value: this.getValue("deliveryDestination.filetype") || 'CSV',
	store: [
		["CSV", "CSV"],
		["XML", "XML"],
		["JSON", "JSON"],
	],
	mode: "local",
	editable: false,
	triggerAction: "all"
}
```
_So, how to add a new filetype in this case_

**Comment by fashxp** [comment]
hi,
please contact your sales contact person at Pimcore to get access to the corresponding github repositories.
there we are happy to help you. thanks.

**Reply by amarjeet6694** [reply]
Hi @fashxp,
Thanks for the reply, will do the same.

**Comment by mathmul** [comment]
How did you @amarjeet6694 even get the `pimcore/data-hub-file-export` if it doesn't exist in [packagist.org](https://packagist.org/?query=pimcore%2Fdata-hub-file-export)?

Following [these instructions](https://pimcore.com/docs/data-hub-file-export/current/Installation/index.html) immediately results in an error.

**Reply by kingjia90** [reply]
it's an [enterprise ](https://pimcore.com/en/platform/enterprise-edition) bundle, which is managed in private repository (repman) that requires subscription/credentials
